packages/mds-transactions-service/tests/transactions.spec.ts

```diff
@@ -0,0 +1,317 @@
+
+          const { transactions: firstPage, cursor: firstCursor } = await TransactionServiceClient.getTransactions({
+            start_timestamp,
+            end_timestamp
+          })
+          expect(firstPage.length).toEqual(10) // default page size
+          firstPage.forEach(({ timestamp }) => {
+            expect(timestamp).toBeGreaterThanOrEqual(start_timestamp)
+            expect(timestamp).toBeLessThanOrEqual(end_timestamp)
+          })
+
+          const { transactions: secondPage } = await TransactionServiceClient.getTransactions({
+            start_timestamp,
+            end_timestamp,
+            after: firstCursor.afterCursor ?? undefined
+          })
+          expect(secondPage.length).toEqual(5)
+          secondPage.forEach(({ timestamp }) => {
+            expect(timestamp).toBeGreaterThanOrEqual(start_timestamp)
+            expect(timestamp).toBeLessThanOrEqual(end_timestamp)
+          })
+        })
+
+        it('Get Bulk Transactions with sort options, with default paging', async () => {
+          const transactionsToPersist = [...transactionsGenerator(20)]
+          await TransactionServiceClient.createTransactions(transactionsToPersist)
+
+          const { transactions: firstPage, cursor: firstCursor } = await TransactionServiceClient.getTransactions({
+            order: {
+              column: 'timestamp',
+              direction: 'DESC'
+            }
+          })
+
+          console.log(firstPage)
+
+          firstPage.reduce<TransactionDomainModel | undefined>((prevTransaction, currTransaction) => {
+            if (prevTransaction) {
+              const { timestamp: prevTimestamp } = prevTransaction
+              const { timestamp: currTimestamp } = currTransaction
+              expect(currTimestamp).toBeLessThanOrEqual(prevTimestamp)
+            }
+
+            return currTransaction
+          }, undefined)
+
+          const { transactions: secondPage } = await TransactionServiceClient.getTransactions({
+            after: firstCursor.afterCursor ?? undefined,
+            order: {
+              column: 'timestamp',
+              direction: 'DESC'
+            }
+          })
+
+          secondPage.reduce<TransactionDomainModel | undefined>((prevTransaction, currTransaction) => {
+            if (prevTransaction) {
+              const { timestamp: prevTimestamp } = prevTransaction
+              const { timestamp: currTimestamp } = currTransaction
+              expect(currTimestamp).toBeLessThanOrEqual(prevTimestamp)
+            }
+
+            return currTransaction
+          }, undefined)
+        })
+      })
+
+      describe('Failure', () => {
+        it('Verify that asking for too many items will fail (i.e. is Joi doing its job)', async () => {
+          try {
+            await TransactionServiceClient.getTransactions({ limit: 10000 })
+            expect('did not happen').toBe('happened')
```

Comment:
Isn't this `expect` always going to fail, since 'did not happen' is never going to be 'happened', or am I misunderstanding how expect and toBe work?

Reply:
Yeah, I think the preferred style here is something like:
```ts
expect(async () => {
  TransactionServiceClient.getTransactions({ limit: 10000 })
}).rejects.toThrow()
```

Reply:
Yup, I need to go and switch this test over to the better styling